client/src/dataset/components/HDF5ParamsForm.tsx

```diff
@@ -21,7 +21,30 @@ const HDF5ParamsForm: React.SFC<MergedProps> = ({
     handleSubmit,
     handleReset,
     onCancel,
+    setFieldValue,
 }) => {
+
+    const isTimeOut = (values.dataset_paths === "" ) ? true : false;
+    const dsPathOptions = values.dataset_paths.split(", ").map(dsPath => ({ key: dsPath, text: dsPath, value: dsPath }));
```

Comment:
Adding the redundant keys needed by the `Dropdown` here is the right approach, compared to the previous one where this was done in the Python code :+1:

`dataset_paths` should be a list, though, see the corresponding comment on the Python part.

Reply:
Thanks, fixed!